doc/classes/Transform3D.xml

```diff
@@ -118,7 +118,7 @@
 			<param index="1" name="angle" type="float" />
 			<description>
 				Returns a copy of this transform rotated around the given [param axis] by the given [param angle] (in radians).
-				The [param axis] must be a normalized vector.
+				The [param axis] must be a normalized vector (see [method Vector3.normalized]). If [param angle] is positive, the basis is rotated clockwise around the axis.
```

Comment:
I dislike the "clockwise" here. As @aaronfranke already mentioned in https://github.com/godotengine/godot/pull/99650#discussion_r1858844284 in context of Godot you'd rather think of it as a counter-clockwise rotation. If we don't want to go into details then I think we should assume the [Godot's Y-up/right-handed coordinate system](https://raw.githubusercontent.com/godotengine/godot-docs/refs/heads/master/tutorials/3d/img/introduction_to_3d_coordinate_systems.webp), and hence use counter-clockwise here (or [the right-hand rule](https://en.wikipedia.org/wiki/Right-hand_rule#A_rotating_body)).

In a different coordinate system sure it could be clockwise but let's assume Godot's... :upside_down_face: (or don't mention CW/CCW at all).

Reply:
Reason I'm mentioning the rotating direction is because only one of these same descriptions did, so it was a matter of consistency for all 3.
I can change it to "counter-clockwise".